Adding Digital Ocean droplet creation & config instructions

etcadinfinitum opened the pull request:
- [x] Create DO account
- [x] Create DO droplet
- [x] Create non-root account on droplet
- [x] Add ssh keys to droplet (after droplet has been created)
- [x] Might also need `ssh-keygen` instructions?
- [x] Test login abilities via SSH

Not sure what else should belong in this section. I think web server config should be written up separately.

etcadinfinitum committed:
added instructions for account creation on Digital Ocean (related to #1)

etcadinfinitum committed:
Adding screenshots for DO account/droplet creation

etcadinfinitum committed:
DO droplet instructions (related to #1)

etcadinfinitum committed:
Added SSH key instructions & account creation instructions

etcadinfinitum committed:
Cleanup & further polishing
Closes #6 (when PR is merged)

Chris-Johnston commented on Server-Creation.md:
```diff
@@ -0,0 +1,131 @@
+  - Your name (individual or team name)
+  - A valid email address
+  - A password for your account
+  - A credit or debit card to link to the account
```
We should mention here that student credits are a thing, and that if you use them it won't cost anything

https://www.digitalocean.com/community/questions/student-discounts

etcadinfinitum replied:
Yes! I will add this

Chris-Johnston commented on Server-Creation.md:
```diff
@@ -0,0 +1,131 @@
+
+## Step 1: Create Your Digital Ocean Droplet
+
+A **Droplet** is a fancy term Digital Ocean (DO) came up with; fundamentally, it is a remote machine that they provide hardware and infrastructure for. A droplet that you create can be used to deploy websites, run bots, build and run large projects, or anything else you want. 
```
```suggestion
A **Droplet** is a fancy term Digital Ocean (DO) came up with; fundamentally, it is a remote virtual machine that they provide hardware and infrastructure for. A droplet that you create can be used to deploy websites, run bots, build and run large projects, or anything else you want. 
```

Chris-Johnston commented on Server-Creation.md:
```diff
@@ -0,0 +1,131 @@
+
+## Step 2: Log Into Your Droplet
+
+After the droplet is created, you will receive an email with credentials for your droplet's `root` account. The email will look something like this:
```
missing screenshot?

etcadinfinitum replied:
I removed that sentence. Good catch. I decided to omit screenshots because the process isn't so complicated as to necessitate it.

Chris-Johnston commented on Server-Creation.md:
```diff
@@ -0,0 +1,131 @@
+
+After the droplet is created, you will receive an email with credentials for your droplet's `root` account. The email will look something like this:
+
+On first login, you will be required to change the password for `root`. Make it something memorable!
```
may be important to note that no characters will be shown when typing in password

etcadinfinitum replied:
Added

Chris-Johnston reviewed (changes requested):
just a few things that I would add

Chris-Johnston committed:
Accept changes for droplet definition

Co-Authored-By: etcadinfinitum <lizzy.presland@gmail.com>

etcadinfinitum committed:
Made changes per Chris' suggestions
Related to issue #1 - setting up server on a cloud service

etcadinfinitum committed:
Merging local changes with website-based commit
I accepted specific changes from Chris' PR review, and forgot to pull before doing more local work :(
